webapp/packages/core-sdk/src/Resource/CachedMapResource.test.ts

```diff
@@ -5,18 +5,20 @@
  * Licensed under the Apache License, Version 2.0.
  * you may not use this file except in compliance with the License.
  */
+import { toJS } from 'mobx';
+
 import { CachedMapResource } from './CachedMapResource';
 import type { ResourceKey } from './ResourceKey';
 import { resourceKeyList } from './ResourceKeyList';
 
-interface ILoaderData {
+interface IEntityData {
   id: string;
   value: number;
 }
 
 const ERROR_ITEM_ID = 'error';
 
-const LOADER_DATA_MOCK: ILoaderData[] = [
+const DATA_MOCK: IEntityData[] = [
   {
     id: '1',
```

Comment:
Please make it's getter so it will return a new object every time. This will help to make tests clear and solve possible problems with data mutation